docs/APP-CONNECTIVITY.md

```diff
@@ -241,6 +260,52 @@ inbound ACL rules are that much more important.
 A metadata HTTP server is run for a switch network instance only if it has a port attached
 that has an IP address.
 
+#### IP address detection
+
+Unlike a Local Network Instance, a switch network instance is configured without any IP
+configuration, and EVE does not run an internal DHCP server. Instead, if IP connectivity
+is required, IP addresses must be assigned statically within the connected applications
+or provided by an external DHCP server or another application offering DHCP services.
+
+Since EVE is not in control of IP address allocations and leases, it must monitor application
```

Comment:
Wow, I didn't know it works that way =)

Reply:
@OhmSpectator for a switch network instance that is the case - EVE provides L2 connectivity but no IP and DNS. For a local network instance it is different.